### DISCUSSION TITLE
Access all of the table data in relational picker component

### DISCUSSION THREAD

**Discussion by TommasoBernini** [Budibase/budibase#7167] [Help] [answered]
Hi everyone,
we are building an admin panel with CRUD functionality using a proprietary Postegres database as data source.

While building the update form to modify the values of a given row, we stumbled upon the problem of linking values from other tables. This has been successfully solved by defining relationships between columns through the row's UUID.

![Schermata 2022-08-09 alle 10 05 03](https://user-images.githubusercontent.com/24413162/183597840-4ad4aa75-8074-415d-af88-3d807464f15b.png)
 
and in the update form using the relationship picker which creates a drop-down list with all of the suitable elements of the linked table. However, all the elements in the list are identified by the non-human readable UUID, the only column which links the data between two rows. 

![Schermata 2022-08-09 alle 10 29 26](https://user-images.githubusercontent.com/24413162/183613602-ba76a80e-39e5-4e32-b509-e9b6a71f053a.png)

I saw an older [discussion](https://github.com/Budibase/budibase/discussions/3690) suggestions the use of formula columns, I was wondering if there were another way to access all of the information inside the linked table in the relationship picker, in order to display instead of the UUID, another value of the linked table, or otherwise another way to achieve the same task with other tools/components.

Thanks for the help :)

**Comment by melohagan** [comment, marked as answer]
Hi @TommasoBernini 

For the Relationship picker, the label and data columns are automatically assigned. 
The label should be the same as the **table display column**.

To change this, edit the column you would like to show as the label for your relationship picker, and toggle the setting on:
![Screenshot 2022-08-09 at 11 39 30](https://user-images.githubusercontent.com/101575380/183628648-3202c604-88ce-4708-ba14-115a308e2e32.png)

**Reply by TommasoBernini** [reply]
Thank you @melohagan, it worked perfectly!
